lib/azure/armrest/wrapper/availability_set.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_wrapper'
+
+module Azure
+  module Armrest
+    class AvailabilitySet < Azure::ArmRest::BaseWrapper
```

Comment:
Initially we use code to generate wrapper classes. For each individual class if we need to add custom method, then it is the time to physically add a new file.